provisioning/transport/amqp/tests/RetryJitterTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Provisioning.Client.Transport;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace Microsoft.Azure.Devices.Provisioning.Transport.Amqp.UnitTests
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class RetryJitterTests
+    {
+        [TestMethod]
+        public void RetryJitterGeneratedDelayLargerOrEqualToDefaultDelay()
```

Comment:
The naming convention is actually <Unit>_<Scenario>_<Expectation>. This way you can read it as a story: `RetryJitter_DefaultDelay_WithinRange`.